src/grids.jl

```diff
@@ -103,13 +103,13 @@ function RegularCartesianGrid(T, N, L)
 
     V = Δx*Δy*Δz
 
-    xC = Δx/2:Δx:Lx
-    yC = Δy/2:Δy:Ly
-    zC = -Lz+Δz/2:Δz:-Δz/2
+    xC = Nx == 1 ? (Δx/2:Δx:Lx) : range(Δx/2, Lx-Δx/2; length=Nx)
```

Comment:
Why the if statement? I get

```julia
julia> a = range(1.0, 1.0, length=1)
1.0:0.0:1.0
```

which seems right.

Reply:
Hmmm, when I first made the change I hit an error like
```julia
julia> range(0.5, 1; length=1)
ERROR: ArgumentError: range(0.5, stop=1.0, length=1): endpoints differ
```
so I decided to reproduce the previous behavior when `N == 1` but maybe I fixed that...

Doing this manually seems to work, so I'll try switching back to `range`.